test/parallel_api/numeric/numeric.ops/inclusive_scan_by_segment.pass.cpp

```diff
@@ -53,31 +53,69 @@ struct test_inclusive_scan_by_segment
         }
     }
 
-    template <typename Iterator1, typename Iterator2, typename Size>
+    template <typename Iterator>
+    int
+    accum_values(Iterator host_vals, int idx_begin, int idx_end)
+    {
+        int summ = 0;
+
+        for (int i = idx_begin; i != idx_end; ++i)
+            summ += host_vals[i];
+
+        return summ;
+    }
+
+#ifdef _ONEDPL_DEBUG_SYCL
```

Comment:
The macro `_ONEDPL_DEBUG_SYCL` may be used in other contexts. And in case of big n, your function for dumping data may lead to the problems.
More preferable approach  - to wrap "dumping data" code into a special local macro, disabled by default.